src/transformStateWithClones.js

```diff
@@ -8,6 +8,30 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const clone = { ...state };
+  const transform = [];
+
+  for (const action of actions) {
```

Comment:
Destructure the action
```suggestion
  for (const { type, extraData, ... } of actions) {
```